.travis.compose.yml

```diff
@@ -24,3 +24,4 @@ services:
       - ./pairing/benches:/usr/src/HoneyBadgerMPC/pairing/benches
       - ./pairing/Cargo.toml:/usr/src/HoneyBadgerMPC/pairing/Cargo.toml
       - ./pairing/setup.py:/usr/src/HoneyBadgerMPC/pairing/setup.py
+      - /usr/src/HoneyBadgerMPC/honeybadgermpc/ntl  # Directory _not_ mounted from host
```

Comment:
We should document this to avoid having to debug this kind of issue again.

Can you open an issue and take that up in a separate PR?

Reply:
Will do.

Reply:
Referenced this issue in #180